consent-ws/src/main/java/org/broadinstitute/consent/http/resources/Resource.java

```diff
@@ -67,6 +72,14 @@ protected Response createExceptionResponse(Exception e) {
                 Response.status(Response.Status.NOT_FOUND).entity(new Error(e.getMessage(), Response.Status.NOT_FOUND.getStatusCode())).build());
         dispatch.put(UpdateConsentException.class, e ->
                 Response.status(Response.Status.BAD_REQUEST).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
+        dispatch.put(SignatureException.class, e->
+                Response.status(Response.Status.BAD_REQUEST).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
+        dispatch.put(MalformedJwtException.class, e->
+                Response.status(Response.Status.BAD_REQUEST).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
+        dispatch.put(UnsupportedJwtException.class, e->
+                Response.status(Response.Status.BAD_REQUEST).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
+        dispatch.put(DecodingException.class, e->
+                Response.status(Response.Status.BAD_REQUEST).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
```

Comment:
I question whether all of these are really bad requests. Some seem like they should at least be internal server errors. Additionally, I think it might make more sense to catch these in `NihServiceAPI` instead and log them since we should be alerted via sentry if we see these kinds of errors.